whitepapers/Tables.md

```diff
@@ -2,4 +2,40 @@
 
 
 
+## Pipeline configuration
 
+### Source
+A Source is a place metadata comes from.
+
+#### Columns
+* `name` -- Short, unique string
+* `long_title` -- full, human-friendly name
```

Comment:
I added a template and changed the rest to follow it, comments/criticisms welcome.